react/features/toolbox/functions.web.js

```diff
@@ -4,6 +4,32 @@ import { hasAvailableDevices } from '../base/devices';
 
 declare var interfaceConfig: Object;
 
+/**
+ * Returns a set of button names to be displayed in the toolbox, based on the screen width.
+ *
+ * @param {number} width - The width of the screen.
+ * @returns {Set} The button set.
+ */
+export function getToolbarAdditionalButtons(width: number): Set<string> {
+    if (width <= 500) {
```

Comment:
you'd get a nicer indentation if you start from the smallest width

Reply:
Indeed, but this is _somehow_ optimised for making less computations on larger screens (which are the main target)